Doc/c-api/unicode.rst

```diff
@@ -307,6 +307,36 @@ These APIs can be used for fast direct character conversions:
    possible.  This function does not raise exceptions.
 
 
+.. c:function:: Py_ssize_t PyUnicode_ToLower(Py_UCS4 ch, Py_UCS4 *buffer)
```

Comment:
I would prefer to have a buffer size parameter and return an error if the buffer is too small.